android/assets/jsons/translations/Indonesian.properties

```diff
@@ -2429,33 +2429,33 @@ in your cities =
 Your = 
 in all coastal cities = di semua kota pesisir
 Coastal = Pesisir
-in capital = di ibu kota
+in capital = di ibukota
```

Comment:
According to the Indonesian dictionary, ibukota is the informal form of ibu kota, so maybe we should just leave it as is?